core/src/avm2/globals/boolean.rs

```diff
@@ -53,6 +53,39 @@ pub fn class_init<'gc>(
     Ok(Value::Undefined)
 }
 
+/// Implements `Boolean.toString`
+pub fn to_string<'gc>(
+    _activation: &mut Activation<'_, 'gc, '_>,
+    this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error> {
+    if let Some(this) = this {
+        if let Some(this) = this.as_primitive() {
+            match this.coerce_to_boolean() {
```

Comment:
FYI, in FP, I'm getting:
```js
var a = Boolean.prototype.toString.apply(1);
```
```
TypeError: Error #1004: Method Boolean.prototype.toString was invoked on an incompatible object.
```
(Not a big immediate deal if you think this should be refactored more widely)

Reply:
The documentation only lists these methods in the `AS3` namespace; I didn't include them in `public`. However, upon further tests it appears overridable `public` versions of these methods exist in on these classes, so I'm going to implement those as well. As part of that I'll also make that particular method throw an error.